### PR TITLE
move to compliance repo

### PR DESCRIPTION
**Pull request by TylerLeonhardt**

### CODE REVIEW

**Review by adityapatwardhan — APPROVED** [review]

File: .vsts-ci/azure-pipelines-release.yml [hunk 1]
```diff
@@ -31,155 +34,138 @@ trigger:
...
+        # The file pattern to use
+        # If not using minimatch: comma separated, with * supported
+        # If using minimatch: newline separated, with !, **, and * supported.
+        # See link in the useMinimatch comments.
+        pattern: 'NStack.dll,Terminal.Gui.dll'
+        # decides if the task should use minimatch for the pattern matching.
+        # https://github.com/isaacs/minimatch#features
+        useMinimatch: false
+
+    - task: PowerShell@1
```
Comment (TravisEz13) [inline .vsts-ci/azure-pipelines-release.yml:123]:
```suggestion
    - task: PowerShell@2
```

Comment (TravisEz13) [reply .vsts-ci/azure-pipelines-release.yml:123]: and really it's better to use the short from

```powershell
-pwsh | 
    my script
  displayName: fake pwsh task
```

File: src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1 [hunk 2]
```diff
@@ -8,9 +8,9 @@
             'publish/NStack.dll'
         )
 
-        'OutGridView.Models' = @(
-            'publish/OutGridView.Models.dll',
-            'publish/OutGridView.Models.pdb'
+        'Microsoft.PowerShell.OutGridView.Models' = @(
+            'publish/Microsoft.PowerShell.OutGridView.Models.dll',
+            'publish/Microsoft.PowerShell.OutGridView.Models.pdb'
```
Comment (adityapatwardhan) [inline src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1:13]: do you publish the PDB too?

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Comment by TylerLeonhardt** [comment]
@TravisEz13 @adityapatwardhan I've moved to the new syntax. Please rereview.

File: src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1 (hunk 2 above)
Comment (TylerLeonhardt) [reply src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1:13]: yeah... why not? :)

File: .vsts-ci/azure-pipelines-release.yml [hunk 3]
```diff
@@ -31,155 +34,127 @@ trigger:
...
+- stage: Build
+  displayName: Build
+  jobs:
+  - job: 'ReleaseBuild'
+    displayName: Release Build
+    pool:
+      vmImage: 'vs2017-win2016'
+    steps:
+    - template: templates/ci-general.yml
+  - job: 'SignBuild'
```
Comment (adityapatwardhan) [inline .vsts-ci/azure-pipelines-release.yml:54]: You need to run component governance here or upload assets.json file from here and download them in compliance stage.

File: .vsts-ci/azure-pipelines-release.yml [hunk 4]
```diff
@@ -31,155 +34,127 @@ trigger:
...
+    name: PowerShell/compliance
+
+stages:
+- stage: Build
+  displayName: Build
+  jobs:
+  - job: 'ReleaseBuild'
+    displayName: Release Build
+    pool:
+      vmImage: 'vs2017-win2016'
```
Comment (adityapatwardhan) [inline .vsts-ci/azure-pipelines-release.yml:51]:
```suggestion
      vmImage: 'windows-latest'
```

File: .vsts-ci/azure-pipelines-release.yml [hunk 5]
```diff
@@ -31,155 +34,127 @@ trigger:
...
+      displayName: 'Upload Artifacts'
+      condition: succeededOrFailed()
+
+- stage: compliance
+  displayName: Compliance
+  dependsOn: Build
+  jobs:
+  - job: Compliance_Job
+    pool:
+      name: Package ES CodeHub Lab E
```
Comment (adityapatwardhan) [inline .vsts-ci/azure-pipelines-release.yml:134]:
```suggestion
      name: Package ES Standard Build
```

File: .vsts-ci/azure-pipelines-release.yml [hunk 6]
```diff
@@ -31,155 +34,127 @@ trigger:
...
+    displayName: Release Build
+    pool:
+      vmImage: 'vs2017-win2016'
+    steps:
+    - template: templates/ci-general.yml
+  - job: 'SignBuild'
+    displayName: Signing Build
+    dependsOn: 'ReleaseBuild'
+    pool:
+      name: 'Package ES CodeHub Lab E'
```
Comment (adityapatwardhan) [inline .vsts-ci/azure-pipelines-release.yml:58]:
```suggestion
      name: 'Package ES Standard Build'
```

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: .vsts-ci/azure-pipelines-release.yml (hunk 3 above)
Comment (TylerLeonhardt) [reply .vsts-ci/azure-pipelines-release.yml:61]: done

**Comment by TylerLeonhardt** [comment]
please rereview @adityapatwardhan

File: src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1 (hunk 2 above)
Comment (TravisEz13) [reply src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1:13]: It's not the nuget standard

**Review by TravisEz13 — APPROVED** [review]

File: src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1 (hunk 2 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1:13]: For information.
From https://devblogs.microsoft.com/dotnet/producing-packages-with-source-link/
> Source Link is easy to add to your projects and we highly recommend that all projects configure it by default.

**Comment by TylerLeonhardt** [comment]
Merging this in now but it might be worth having a discussion around pdbs with the team. Most things I worked on shipped pdbs with it.

**Pull request merged**